providers/rethinkdb.js

```diff
@@ -1,11 +1,11 @@
-const { Provider } = require("klasa");
+const { Provider, util: { mergeObjects } } = require("klasa");
 const rethink = require("rethinkdbdash");
 
 module.exports = class extends Provider {
 
     constructor(...args) {
         super(...args);
-        this.db = rethink(this.client.options.providers.rethinkdb || { db: "pengubot" });
+        this.db = rethink(this.client.options.providers.rethinkdb || { db: "pengubot", timeout: 100 });
```

Comment:
You should revert this change and set up the database configuration in your `KlasaClientOptions`:

```javascript
new KlasaClient({
    // ...
    providers: {
        default: 'rethinkdb',
        rethinkdb: { db: 'pengubot', timeout: 100 }
    }
    // ...
});
```

Reply:
Okay will do, thanks.